README.md

```diff
@@ -149,33 +149,27 @@ Open-source clients to the plotly.js APIs are available at these links:
 |**MATLAB**| [plotly/matlab-api](https://github.com/plotly/matlab-api) | [plotly/matlab/getting-started](https://plotly.com/matlab/getting-started) |
 |**node.js / Tonicdev / Jupyter notebook**| [plotly/plotly-notebook-js](https://github.com/plotly/plotly-notebook-js) | |
 
-## Maintainers
-
-### Active
-
-|   | GitHub | Twitter |
-|---|--------|---------|
-|**Alex C. Johnson**| [@alexcjohnson](https://github.com/alexcjohnson) | |
-|**Antoine Roy-Gobeil** | [@antoinerg](https://github.com/antoinerg) | |
-|**Mojtaba Samimi** | [@archmoj](https://github.com/archmoj) | [@solarchvision](https://twitter.com/solarchvision) |
-|**Nicolas Kruchten** | [@nicolaskruchten](https://github.com/nicolaskruchten) | [@nicolaskruchten](https://twitter.com/nicolaskruchten) |
-
-### Hall of Fame
-
-|   | GitHub | Twitter |
-|---|--------|---------|
-|**Étienne Tétreault-Pinard**| [@etpinard](https://github.com/etpinard) | [@etpinard](https://twitter.com/etpinard) |
-|**Mikola Lysenko**| [@mikolalysenko](https://github.com/mikolalysenko) | [@MikolaLysenko](https://twitter.com/MikolaLysenko) |
-|**Ricky Reusser**| [@rreusser](https://github.com/rreusser) | [@rickyreusser](https://twitter.com/rickyreusser) |
-|**Dmitry Yv.** | [@dy](https://github.com/dy) | [@DimaYv](https://twitter.com/dimayv)|
-|**Robert Monfera**| [@monfera](https://github.com/monfera) | [@monfera](https://twitter.com/monfera) |
-|**Robert Möstl** | [@rmoestl](https://github.com/rmoestl) | [@rmoestl](https://twitter.com/rmoestl) |
-|**Nicolas Riesco**| [@n-riesco](https://github.com/n-riesco) | |
-|**Miklós Tusz**| [@mdtusz](https://github.com/mdtusz) | [@mdtusz](https://twitter.com/mdtusz)|
-|**Chelsea Douglas**| [@cldougl](https://github.com/cldougl) | |
-|**Ben Postlethwaite**| [@bpostlethwaite](https://github.com/bpostlethwaite) | |
-|**Chris Parmer**| [@chriddyp](https://github.com/chriddyp) | |
-|**Alex Vados**| [@alexander-daniel](https://github.com/alexander-daniel) | |
+## Notable Contributors
+
+|   | GitHub | Twitter | Status |
+|---|--------|---------|--------|
+|**Mojtaba Samimi** | [@archmoj](https://github.com/archmoj) | [@solarchvision](https://twitter.com/solarchvision) | Active, Maintainer |
```

Comment:
No. This is rather confusing. I am sure I should not be listed on top of this list.
Let's maintain the `Hall of Fame`.

Reply:
<img width="923" alt="Screen Shot 2020-08-13 at 4 28 14 PM" src="https://user-images.githubusercontent.com/2678795/90183764-0ce45900-dd82-11ea-9d47-f24579e43b1e.png">
@archmoj I think the contributor graphs make it pretty clear that you're the primary maintainer of this project since @etpinard's departure (and arguably on an equal footing to him for some time prior to that). So there should be no question that this placement is deserved. But if you would prefer to install me as a figurehead public face of the project I guess I won't protest.

Reply:
that was my reasoning as well, but I'm happy to switch out the top two

Reply:
Order switched.